content/en/docs/Kyverno CLI/_index.md

```diff
@@ -1052,6 +1059,104 @@ skipped mutate policy add-default-resources -> resource default/Pod/nginx-demo2
 Test Summary: 2 tests passed and 0 tests failed
 ```
 
+In the following policy test, a `generate` policy rule is applied which generates a new resource from an existing resource present in resource.yaml. To test such policy the `generatedResource` tag is applied in `results[]` array, where the new generated resource is added.
```

Comment:
This line could be refactored - To test `generate` policy, the addition of `generatedResource` field in the `results[]` array is must which is used to test against the resource generated by the policy.